issuesync.rb

```diff
@@ -80,9 +80,11 @@ def format(io, issue, comments)
     end
 
     def format_body(io, issue)
-      title = "##{issue.number}: #{issue.title.strip}"
-      title << issue.labels
-      title << "  [CLOSED]" if issue.closed?
+      title = "##{issue.number}: #{issue.title.strip} "
```

Comment:
Assuming we want two spaces after the title, but only one between each label/closed, then how would you recommend doing it? `title = title.strip` might be the easiest approach, to clean up the extra space if we turn out not to use it.